Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.20
+FROM --platform=${TARGETPLATFORM} golang:1.20
```

Comment:
I tested that golang images are available for all platforms currently supported - `linux/arm64`, `linux/s390x` and `linux/amd64`